Firestore/core/test/firebase/firestore/model/field_value_test.cc

```diff
@@ -133,32 +133,30 @@ TEST(FieldValueTest, DeletesHandleMissingKeys) {
 }
 
 TEST(FieldValueTest, DeletesNestedKeys) {
-  FieldValue::Map orig =
-      Object("a", Object("b", 1, "c", Object("d", 2, "e", 3))).object_value();
-  ObjectValue old = WrapObject(orig);
+  ObjectValue old =
+      Object("a", Object("b", 1, "c", Object("d", 2, "e", 3)));
   ObjectValue mod = old.Delete(Field("a.c.d"));
 
   EXPECT_NE(mod, old);
-  EXPECT_EQ(WrapObject(orig), old);
 
-  FieldValue::Map second =
-      Object("a", Object("b", 1, "c", Object("e", 3))).object_value();
-  EXPECT_EQ(WrapObject(second), mod);
+  ObjectValue second =
```

Comment:
For better or worse, this test wanted to have the contents of object on the side to be able to recreate it:

https://github.com/firebase/firebase-android-sdk/blob/aaf3b851ec51134f0e33590136ab3e226545eab1/firebase-firestore/src/test/java/com/google/firebase/firestore/model/FieldValueTest.java#L187

In the Android port this _is_ different because this is going to go through the `UserDataConverter` to actually produce the result. However this is in the same spirit and I already trust immutable:::SortedMap to be immutable.

Anyway, I don't feel super strongly about this, but this is introducing a divergence.